website/docs/r/monitor_activity_log_alert.html.markdown

```diff
@@ -96,6 +96,15 @@ A `criteria` block supports the following:
 * `recommendation_type` - (Optional) The recommendation type of the event. It is only allowed when `category` is `Recommendation`.
 * `recommendation_category` - (Optional) The recommendation category of the event. Possible values are `Cost`, `Reliability`, `OperationalExcellence` and `Performance`. It is only allowed when `category` is `Recommendation`.
 * `recommendation_impact` - (Optional) The recommendation impact of the event. Possible values are `High`, `Medium` and `Low`. It is only allowed when `category` is `Recommendation`.
+* `servicehealth` - (Optional) A block to define fine grain service health settings.
```

Comment:
This is supposed to be `service_health`.

Reply:
I will update the docs

Reply:
Are these changes deployed ? if it is yes then how can I use this fine granular piece of code ? Because right now it is giving error "Blocks of type "servicehealth" are not expected here."

Reply:
@skyflash711 no it's an PR er have to wait till it's reviewed and merged.

After that, a comment from the bot will be added with the version where feature is part of.